packages/icons/src/arrow-icon.tsx

```diff
@@ -1,27 +0,0 @@
-import { memo } from 'react'
-
-import { themed } from '@status-im/icons/src/themed'
```

Comment:
Removed, because this `themed` does not exists and it seems like the components were moved to `src/icons`.